beacon-chain/sync/validate_aggregate_proof.go

```diff
@@ -222,7 +222,7 @@ func (s *Service) validateAggregatedAtt(ctx context.Context, signed ethpb.Signed
 	return s.validateWithBatchVerifier(ctx, "aggregate", set)
 }
 
-func (s *Service) validateBlockInAttestation(ctx context.Context, satt ethpb.SignedAggregateAttAndProof) bool {
+func (s *Service) validateBlockPresenceOrQueueAttestation(ctx context.Context, satt ethpb.SignedAggregateAttAndProof) bool {
```

Comment:
Shorter name + documentation would be better IMO

Reply:
honestly it feels like there's a deeper issue here why is the save like hacked into this lol the function does 2 things.

Reply:
In that case, should we not separate it into two different functions?